centreon-connector/common/inc/com/centreon/connector/parser.hh

```diff
@@ -0,0 +1,76 @@
+/*
+** Copyright 2011-2013 Centreon
```

Comment:
This file looks new :smile: 
So, we can make the copyright to 2022

Reply:
In fact, it is just moved..., so we just have to complete the date

Reply:
done